pyqtgraph/graphicsItems/ROI.py

```diff
@@ -1914,15 +1926,23 @@ def shape(self):
         p.lineTo(self.handles[0]['item'].pos())
         return p
 
-    def getArrayRegion(self, data, img, axes=(0,1), **kwds):
+    def getArrayRegion(self, data, img, axes=(0,1),
+                       returnMappedCoords=False, **kwds):
         """
         Return the result of ROI.getArrayRegion(), masked by the shape of the 
         ROI. Values outside the ROI shape are set to 0.
         """
         br = self.boundingRect()
         if br.width() > 1000:
             raise Exception()
-        sliced = ROI.getArrayRegion(self, data, img, axes=axes, fromBoundingRect=True, **kwds)
+        if returnMappedCoords:
+            sliced, mappedCoords = ROI.getArrayRegion(self, data, img,
+                                                      axes=axes,
+                                                      fromBoundingRect=True,
+                                                      **kwds)
```

Comment:
do you need `returnMappedCoords=True` here?

Reply:
You are right. This is utterly wrong. I got confused and I mixed up the use of  *fromBoundingRect* and *returnMappedCoords*.

I pushed a fix.